examples/outputs/foundry.rs

```diff
@@ -179,15 +179,15 @@ async fn main() -> Result<()> {
     // send native token
     //////////////////////////////////
 
-    let basic_output_builder = BasicOutputBuilder::new_with_amount(1_000_000)?
+    let basic_output_builder = BasicOutputBuilder::new_with_amount(57_700)?
```

Comment:
Just curious: why?

Reply:
I couldn't get the example to run using higher amounts.  If you think I was getting the error due to something else I can revert this change.